src/prebid.js

```diff
@@ -481,26 +485,25 @@ $$PREBID_GLOBAL$$.clearAuction = function() {
  * @param adUnitCodes
  */
 $$PREBID_GLOBAL$$.requestBids = function ({ bidsBackHandler, timeout, adUnits, adUnitCodes }) {
+  timeout = timeout || $$PREBID_GLOBAL$$.bidderTimeout;
+  adUnits = adUnits || $$PREBID_GLOBAL$$.adUnits;
+
+  // if specific adUnitCodes filter adUnits for those codes
+  if (adUnitCodes && adUnitCodes.length) {
+    adUnits = adUnits.filter(adUnit => adUnitCodes.includes(adUnit.code));
+  }
+
   if (auctionRunning) {
-    utils.logError('Prebid Error: `$$PREBID_GLOBAL$$.requestBids` was called while a previous auction was' +
-      ' still running. Resubmit this request.');
+    bidRequestQueue.push(() => {
+      $$PREBID_GLOBAL$$.requestBids({ bidsBackHandler, timeout, adUnits });
+    });
     return;
   } else {
     auctionRunning = true;
     $$PREBID_GLOBAL$$._bidsRequested = [];
     $$PREBID_GLOBAL$$._bidsReceived = [];
```

Comment:
This line is causing issues. If you queue 2 bid requests back to back, the 2nd one will clear the first before the adserver returns and finds the winning bids. It might make sense to not clear this at all...